packages/devtools_app/test/test_infra/flutter_test_storage.dart

```diff
@@ -8,15 +8,15 @@ import 'package:devtools_app/src/primitives/storage.dart';
 ///
 /// This ephemeral implementation is meant to help keep unit tests segregated
 class FlutterTestStorage implements Storage {
-  late final Map<String, dynamic> _values = {};
+  late final Map<String, dynamic> values = {};
```

Comment:
this doesn't need to be public since it can be accessed through getValue and setValue

Reply:
It is test storage, not prod storage and I want to be able to check number of values.